### DISCUSSION TITLE
aftermath/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#48] [Announcements]
# aftermath/

After Action Report

https://www.supergoodcode.com/aftermath/

**Comment by DragoonAethis** [comment]
Dunno about these broken colors, I kinda dig the Fifth Elementesque aesthetic tbh

**Reply by tsukasa1234567** [reply]
Maybe it was just lacking it's Multi-Pass, or it didn't go through the Auto-Wash?
